<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Priya" failures="0" tests="2" name="Extent_Report.Skipped_Status" time="1.028" errors="1" timestamp="2024-07-29T12:23:14 IST" skipped="1">
  <testcase classname="Extent_Report.Skipped_Status" name="deleteAccount" time="0.000">
    <skipped/>
  </testcase> <!-- deleteAccount -->
  <testcase classname="Extent_Report.Skipped_Status" name="createAccount" time="1.028">
    <error message="Method Extent_Report.Skipped_Status.createAccount() didn&#039;t finish within the time-out 1000" type="org.testng.internal.thread.ThreadTimeoutException">
      <![CDATA[org.testng.internal.thread.ThreadTimeoutException: Method Extent_Report.Skipped_Status.createAccount() didn't finish within the time-out 1000
at java.base/java.lang.Thread.sleepNanos0(Native Method)
at java.base/java.lang.Thread.sleepNanos(Thread.java:491)
at java.base/java.lang.Thread.sleep(Thread.java:522)
at org.apache.commons.exec.DefaultExecuteResultHandler.waitFor(DefaultExecuteResultHandler.java:121)
at org.openqa.selenium.os.OsProcess.waitFor(OsProcess.java:165)
at org.openqa.selenium.os.CommandLine.waitFor(CommandLine.java:108)
at org.openqa.selenium.manager.SeleniumManager.runCommand(SeleniumManager.java:116)
at org.openqa.selenium.manager.SeleniumManager.getDriverPath(SeleniumManager.java:289)
at org.openqa.selenium.remote.service.DriverFinder.getPath(DriverFinder.java:22)
at org.openqa.selenium.remote.service.DriverFinder.getPath(DriverFinder.java:13)
at org.openqa.selenium.chrome.ChromeDriver.generateExecutor(ChromeDriver.java:99)
at org.openqa.selenium.chrome.ChromeDriver.<init>(ChromeDriver.java:88)
at org.openqa.selenium.chrome.ChromeDriver.<init>(ChromeDriver.java:83)
at org.openqa.selenium.chrome.ChromeDriver.<init>(ChromeDriver.java:51)
at Extent_Report.Skipped_Status.createAccount(Skipped_Status.java:14)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.InvokeMethodRunnable.runOne(InvokeMethodRunnable.java:47)
at org.testng.internal.invokers.InvokeMethodRunnable.call(InvokeMethodRunnable.java:76)
at org.testng.internal.invokers.InvokeMethodRunnable.call(InvokeMethodRunnable.java:11)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1570)
]]>
    </error>
  </testcase> <!-- createAccount -->
</testsuite> <!-- Extent_Report.Skipped_Status -->
